ts/DataGrid/DataGrid.ts

```diff
@@ -937,6 +943,55 @@ class DataGrid {
         DataGrid.dataGrids.splice(dgIndex, 1);
     }
 
+    /**
+     * Grey out the data grid and show a loading indicator.
+     *
+     * @param message
+     * The message to display in the loading indicator.
+     */
+    public showLoading(message?: string): void {
+        // Create loading wrapper.
+        this.loadingWrapper = makeHTMLElement(
+            'div',
+            {
+                className: 'highcharts-datagrid-loading-wrapper',
```

Comment:
Could you add move the class names to `Globals.ts`? Valid also for the 966 and 976 lines.